File: Examples/CSharp/Images/ConcatTIFFImages.cs
using System.IO;

using Aspose.Imaging;
using Aspose.Imaging.FileFormats.Tiff;

namespace Aspose.Imaging.Examples.Images
{
    public class ConcatTIFFImages
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = Aspose.Imaging.Examples.Utils.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);

            //Create a copy of original image to avoid any alteration
            File.Copy(dataDir + "demo.tif", dataDir + "TestDemo.tif", true);

            //Create an instance of TiffImage and load the copied destination image
            using (TiffImage image = (TiffImage)Aspose.Imaging.Image.Load(dataDir + "TestDemo.tif"))
            {
                //Create an instance of TiffImage and load the source image
                using (TiffImage image1 = (TiffImage)Aspose.Imaging.Image.Load(dataDir + "sample.tif"))
                {
                    // Create an instance of TIffFrame and copy active frame of source image
                    TiffFrame frame = TiffFrame.CopyFrame(image1.ActiveFrame);
                    
                    // Add copied frame to destination image
                    image.AddFrame(frame);
                    
                    // save the image with changes.
                    image.Save();
                }
            }

            // Display Status.
            System.Console.WriteLine("Concatenation of TIF files done successfully.");
        }
    }
}

File: Examples/VisualBasic/Images/ConcatTIFFImages.vb
Imports System.IO

Imports Aspose.Imaging
Imports Aspose.Imaging.FileFormats.Tiff

Namespace Aspose.Imaging.Examples.VisualBasic.Images
    Public Class ConcatTIFFImages
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Images("ConcatTIFFImages")

            'Create a copy of original image to avoid any alteration
            File.Copy(dataDir & "demo.tif", dataDir & "TestDemo.tif", True)

            'Create an instance of TiffImage and load the copied destination image
            Using image As TiffImage = CType(Global.Aspose.Imaging.Image.Load(dataDir & "TestDemo.tif"), TiffImage)
                'Create an instance of TiffImage and load the source image
                Using image1 As TiffImage = CType(Global.Aspose.Imaging.Image.Load(dataDir & "sample.tif"), TiffImage)
                    ' Create an instance of TIffFrame and copy active frame of source image
                    Dim frame As TiffFrame = TiffFrame.CopyFrame(image1.ActiveFrame)

                    ' Add copied frame to destination image
                    image.AddFrame(frame)

                    ' save the image with changes.
                    image.Save()
                End Using
            End Using

            ' Display Status.
            System.Console.WriteLine("Concatenation of TIF files done successfully.")
        End Sub
    End Class
End Namespace